### PR TITLE
Change "user_preferences.inc.php" file as a class

### PR DESCRIPTION
**Pull request by Tithugues**
### Description

To avoid including files that will do some code directly, I replaced the `user_preferences.inc.php` file by a class and used it.

Fixes #

Before submitting pull request, please review the following checklist:

- [x] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [x] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [x] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [x] Every commit has a descriptive commit message.
- [x] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [x] Any new functionality is covered by tests.

### CODE REVIEW

File: libraries/classes/UserPreferencesHeader.php [hunk 1]
```diff
@@ -0,0 +1,136 @@
+<?php
+/* vim: set expandtab sw=4 ts=4 sts=4: */
+/**
+ * Holds the PhpMyAdmin\UserPreferences class
```
Comment (williamdes) [inline libraries/classes/UserPreferencesHeader.php]: did you mean `UserPreferencesHeader` ?

**Review by williamdes — APPROVED** [review]
LGTM, thanks for contributing !

File: libraries/classes/UserPreferencesHeader.php (hunk 1 above)
Comment (Tithugues) [reply libraries/classes/UserPreferencesHeader.php]: Yes, thanks for noticing it. (I duplicated the `UserPreferencesTest` class…)

Comment (Tithugues) [reply libraries/classes/UserPreferencesHeader.php]: This should be fixed in cc8c037.

**Review by MauricioFauth — APPROVED** [review]

**Comment by mauriciofauth** [comment]
Merged, thanks for your contribution.

**Pull request merged**